src/modules/maps/containers/MapVisuals.jsx

```diff
@@ -7,8 +7,8 @@ import { MapHelper } from '../../../helpers/mapexplorer.js';
 import { enableViewCameraMode } from '../actions/mapexplorer';
 
 const mapconfig = require('../../../constants/mapExplorer.config.json');
-const gorongosaGeodata = require('../../../map-data/gorongosa-geodata.json');
-const vegetationGeodata = require('../../../map-data/vegetation-geodata.json');
+const gorongosaGeodata = require('../data/gorongosa-geodata.json');
+const vegetationGeodata = require('../data/vegetation-geodata.json');
```

Comment:
Ah, I understand what you mean now! Thanks for that spot, @simoneduca, I've just fixed MapLegendVegetation. Not sure why my code compiled properly last week when it shouldn't have.